lib/scala/bench-processor/src/main/java/org/enso/benchmarks/processor/BenchProcessor.java

```diff
@@ -231,7 +231,7 @@ private void generateClassForGroup(
         "      throw new IllegalStateException(\"Project root directory does not exist or cannot be read: \" + Objects.toString(projectRootDir));");
     out.println("    }");
     out.println("    File languageHomeOverride = Utils.findLanguageHomeOverride();");
-    out.println("    var ctx = Context.newBuilder(LanguageInfo.ID)");
+    out.println("    var ctx = Context.newBuilder()");
```

Comment:
I thought it is going to be classpath issue, not "restricted to Enso language" issue! Thank you for your help, @Akirathan!